packages/babel-preset-react/README.md

```diff
@@ -64,3 +79,11 @@ require("babel-core").transform("code", {
   presets: ["react"]
 });
 ```
+
+## Options
+
+### `development`
```

Comment:
I think we can explain here/somewhere that you can use this in another "env" either via "env" config in babelrc (although we didn't deprecate yet), or via babelrc.js